src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Hover/VSHover.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#nullable enable
+
+using HoverModel = OmniSharp.Extensions.LanguageServer.Protocol.Models.Hover;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Hover
+{
+    internal class VSHover : HoverModel
+    {
+        public object? RawContent { get; set; }
```

Comment:
I assume it's named like this because that's how VS expects it to be shaped? Not a huge fan of this name but if that's the case there's nothing to be done about it for now.

Reply:
Yep! Not a huge fan either but that's how VS names it.